c/meterpreter/source/metsrv/server_transport_winhttp.c

```diff
@@ -290,10 +290,9 @@ static DWORD validate_response_winhttp(HANDLE hReq, HttpTransportContext* ctx)
  */
 static DWORD packet_transmit_http(Remote *remote, LPBYTE rawPacket, DWORD rawPacketLength)
 {
-	DWORD res = 0;
+	DWORD result = ERROR_SUCCESS;
```

Comment:
```suggestion
	DWORD dwResult = ERROR_SUCCESS;
```

Might be good to change this as it provides compatibility with macros in `common.h`.

Reply:
I added this and the common.h macro suggested below. I don't often pull request, so please let me know if I was supposed to hit "Commit suggestion" or something for future reference. Just was unsure since it required more changes then just that line.

Reply:
:+1: You did what I would do. When there's more changes I prefer folks make them manually so they can be contained in a single commit. Thanks alot! The changes look good now so I'm getting started on testing them.